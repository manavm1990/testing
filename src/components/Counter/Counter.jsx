import PropTypes from "prop-types";

function Counter({ count }) {
  return <p className="font-semibold text-lg text-blue-700">{count}</p>;
}

Counter.propTypes = {
  count: PropTypes.number.isRequired,
};

export default Counter;
