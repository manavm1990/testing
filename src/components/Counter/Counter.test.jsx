import React from "react";
import renderer from "react-test-renderer";
import Counter from "./Counter";

describe("My Test Suite", () => {
  it("My Test Case", () => {
    expect(true).toBe(true);
  });
});

describe("Counter", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Counter count={1} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
