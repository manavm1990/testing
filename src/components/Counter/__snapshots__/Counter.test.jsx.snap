// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<p
  className="font-semibold text-lg text-blue-700"
>
  1
</p>
`;
